actionpack/lib/action_controller/metal/exceptions.rb

```diff
@@ -22,6 +22,12 @@ def initialize(message, failures = [])
     end
   end
 
+  class MissingExactTemplate < ActionControllerError #:nodoc:
+    def initialize(message)
```

Comment:
If there isn't any additional work to be done here other than calling `super`, this block can be omitted, no? (Just like the one below)

Reply:
as far as I understood, the message is passed to the exception, therefore the super is called

Reply:
`MissingExactTemplate` inherits from `ActionControllerError` which inherits `StandardError`. `StandardError` already has an initializer accepting just a message argument. Without your overridden initializer you could still initialize `MissingExactTemplate` the same way you would a `StandardError` because it inherits it's initializer from `StandardError`.

In summary your initializer is only calling its super's initializer and is equivalent (in this case) to not defining an initializer.

Reply:
Yeah, you can remove this initialize.